tests/subsys/power/power_states_api/src/main.c

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2020 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <ztest.h>
+#include <power/power.h>
+#include <devicetree/pm_state.h>
```

Comment:
We don't do it  like this. The files in include/devicetree/ are private headers which we include from the bottom of devicetree.h; please keep this consistent.

Reply:
good point, will change it